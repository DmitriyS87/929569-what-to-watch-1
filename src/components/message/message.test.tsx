import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Message from './message';

it(`Message correctly render text:`, () => {
  const tree = renderer.create(
    <Message
      text={`is correct message`}
      style={``}
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
