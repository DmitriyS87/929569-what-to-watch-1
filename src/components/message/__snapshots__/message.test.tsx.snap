// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message correctly render text: 1`] = `
<div
  className=""
>
  <p>
    is correct message
  </p>
</div>
`;
